framework/filters/HttpCache.php

```diff
@@ -182,6 +182,10 @@ protected function validateCache($lastModified, $etag)
      */
     protected function sendCacheControlHeader()
     {
+        if (headers_sent($file, $line)) {
```

Comment:
Is this check really necessary? Would not PHP throw its own error saying 'headers have already been sent'?